File: src/SqlCrawler.Backend/Core/SqlServerInfo.cs
namespace SqlCrawler.Backend.Core
{
    public class SqlServerInfo: SqlServerInfoPublic
    {
        public string DataSource { get; set; }
        public bool UseIntegratedSecurity { get; set; }
        public string SqlUsername { get; set; }
        public string SqlPassword { get; set; }
    }
}

File: src/vue/src/cs-core/SqlServerInfo.ts
import { sqlServerInfoPublic } from './SqlServerInfoPublic';

export class sqlServerInfo extends sqlServerInfoPublic {
    public dataSource: string | null = null;
    public useIntegratedSecurity: boolean | null = false;
    public sqlUsername: string | null = null;
    public sqlPassword: string | null = null;
}
